### ISSUE TITLE
fwLandingPattern added by QGC but unsupported by MAVSDK

### ISSUE THREAD

**Issue by theloni-monk** [mavlink/MAVSDK#1884]
See: https://github.com/mavlink/MAVSDK-Python/issues/508#issuecomment-1217406640

Currently it is impossible to export a plan for a fixed-wing aircraft from QGC and upload the .plan file via the MAVSDK framework. This implementation would be greatly appreciated.

**Comment by julianoes** [comment]
Yes, it is possible to import it into MissionRaw. Mission however doesn't support fixedwing.
